app/src/main/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentPresenter.kt

```diff
@@ -26,6 +26,9 @@ class ProfileProgressFragmentPresenter @Inject constructor(
   @Inject
   lateinit var viewModel: ProfileProgressViewModel
 
+  @Inject
+  lateinit var multiTypeAdapterBuilder: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
This (& technically the view model above) should be injected via the constructor, not field injection.